internal/server/server.go

```diff
@@ -83,6 +84,7 @@ func New(logger Logger, client Client, templates template.Templates, prefix, sir
 	mux.Handle("/allocate-cases", wrap(AllocateCases(client, templates.Get("allocate_cases.gohtml"))))
 	mux.Handle("/assign-task", wrap(AssignTask(client, templates.Get("assign_task.gohtml"))))
 	mux.Handle("/mi-reporting", wrap(MiReporting(client, templates.Get("mi_reporting.gohtml"))))
+	mux.Handle("/get-payments", wrap(GetPayments(client, templates.Get("payments.gohtml"))))
```

Comment:
I think `/payments` would be a better URL, but we probably need to revisit _all_ of these at some point (e.g. I'd eventually like it to be `/case/%d/payments`).